### ISSUE TITLE
`tldr --list` output should be not in a single line

### ISSUE THREAD

**Issue by FazleArefin** [tldr-pages/tldr-node-client#436]
## Expected behavior
Output should be separated by newlines just like the official rust client.

## Actual behavior
Output is in a single line string separated by commas. This makes it difficult to `grep` for a particular command of interest (unless grep complex regex is used). `tldr --list` should output available commands separated by newlines just like the official rust client.

## Log, debug output
NA

## Environment
- Operating system - Linux
- Node.js version v21.6.2
- tldr-node-client version v3.4.0
